lib/prawn-rails-forms.rb

```diff
@@ -3,13 +3,36 @@
 
 module PrawnRailsForms
   class FieldRow
-    attr_accessor :height, :units, :x, :y, :unit_width
-    
-    def initialize(height, units, x, y, unit_width)
-      @height, @units, @x, @y, @unit_width = 
-        height, units, x, y, unit_width
+    attr_accessor :document, :height, :units, :x, :y, :unit_width
+
+    def initialize(document,height, units, x, y, unit_width)
+      @document, @height, @units, @x, @y, @unit_width = 
```

Comment:
`@document` becomes an attribute of the `FieldRow` class, to which the `make_*_field` methods are sent.